standards/development_language_standards.md

```diff
@@ -0,0 +1,104 @@
+  cost of ownership of the service, including exit and transition costs
+- making all new source code open by default
+- using open standards, complying with any that are compulsory for use in government, unless you've been granted an
+  exemption
+
+The simplest way to achieve this is by considering open platforms first.
+ 
+### Our primary development platforms are Microsoft's .Net framework and Node.js
+Defra is engaging on a two-fold approach to its IT application architecture: the use of common commodity products and
+the development of highly user-centric online digital services.
+
+This creates a need to be able to engage with both commercial products, which are often based around Microsoft's
+technology stack, and the open development community.
+
+By focusing on these two key technologies, we benefit from the flexibility that it provides whilst also constraining
+diversity to a level that is appropriate for a medium-sized team of developers.
+
+Other technology options are available, but .Net (specifically C#) and Node.js are the two platforms that fit the need
+best whilst also already having sufficient support in the Defra software development teams.
+
+Because some aspects of certain problems require a small use of additional technologies or because of ongoing need to
+support an existing service, DDTS still needs to maintain some limited capability in a number of other technologies,
+such as Java, R, Python, Ruby, VBA, etc, but .Net and Node.js are preferred.
+ 
+### We develop online digital services in Node.js
+The online digital development community is largely comprised of developers from the open source community, who produce
+and share many freely available frameworks, libraries and tools.
+
+Consequently, development of this kind can best achieve the necessary levels of agility by making use of those same
+technologies.
+This can be seen across government as well, in particular the GDS front end community tend to publish their products
+so that they can be easily used in Ruby and Node.js, in particular.
+ 
+### We use the Hapi framework to develop in Node.js
+Hapi is already in use in Defra digital services and has provided a productive level of standardisation across
+development teams and has proven to be robust and reliable.
+ 
+### We extend commodity platforms by building bespoke capability in such a way that it can be surfaced in the products
+Although our target enterprise architecture is based around the use of commodity platforms, an equally important part
+of that architecture is the part played by bespoke capabilities that those platforms do not provide, or could not
+provide without inappropriate levels of customisation.
+
+Where this is the case, we build that capability in such a way that it can be surfaced within those commodity products,
+either as back end services or "at the glass" integration.
```

Comment:
I can take a guess at what  "at the glass" integration means but it's not a phrase I'm familiar with. Apart from that all looks good to me.